lib/offline/indexeddb/v1_storage_cell.js

```diff
@@ -73,6 +73,7 @@ shaka.offline.indexeddb.V1StorageCell = class
     const V1StorageCell = shaka.offline.indexeddb.V1StorageCell;
 
     return {
+      creationTime: old.creationTime || 0,
```

Comment:
This won't exist in the old format, so you can just fill in 0 or NaN.  After you rebase, you'll need this in v2_storage_cell as well.  We've moved to a new database format.

Reply:
Changed! Thanks!